Packs/AzureFirewall/Integrations/AzureFirewall/AzureFirewall_description.md

```diff
@@ -1,24 +1,60 @@
-## Authenticating
-In order to connect to the Azure Firewall, follow these steps:
+In order to connect to the Azure Firewall use one of the following methods:
 
-1. In the Azure Portal, add a new Azure App Registration. To add the registration, refer to the following [Microsoft article](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app).
-2. Add the following permissions to your registered app:
-   - `Azure Service Management/user_impersonation`
-   - `Microsoft Graph/User.Read`
-3. In your registered app - Get the Application (client) ID. 
-   1. In the Azure Portal, navigate to **App registrations** > your registered application > **Overview**.
-   2. Copy and save the Application (client) ID.
-4. In the *Client ID* parameter, enter your registered app Application (client) ID.
+1. *Client Credentials Flow*
+2. *Device Code Flow*.
+3. *Azure Managed Identities Flow*.
```

Comment:
I recommend the following:
1. We use bullets rather than numbers, as it can be confusing. 
2. Remove the period from the other options, so its "Device Code Flow", etc. 
3. Move this to after the Self-Deployed Azure App section. Its a bit confusing here.